src/services/tx/tx-sender/dispatch.ts

```diff
@@ -115,12 +115,11 @@ export const dispatchTxExecution = async (
   const sdkUnchecked = await getUncheckedSafeSDK(provider)
   const eventParams = txId ? { txId } : { groupKey: await sdkUnchecked.getTransactionHash(safeTx) }
 
-  txDispatch(TxEvent.EXECUTING, eventParams)
-
   // Execute the tx
   let result: TransactionResult | undefined
   try {
     result = await sdkUnchecked.executeTransaction(safeTx, txOptions)
+    txDispatch(TxEvent.EXECUTING, eventParams)
```

Comment:
So basically if I don't confirm or reject the execution, this won't be called?

Reply:
Yes. But I think the subscribing problem still persists. After the refresh I don't see the "Submitting" state

Reply:
Well, at least it will pick it up from the history if it's executed. Or not if it's rejected.